docs/expo.md

```diff
@@ -0,0 +1,182 @@
+
+- [Guide for Expo apps](#guide-for-expo-apps)
+- [Guide for bare React Native apps using Expo modules](#guide-for-bare-react-native-apps-using-expo-modules)
+
+# Guide for Expo apps
+
+## Overview
+Expo projects can use both third-party React Native libraries with native code and custom native code. Creating a development build with Expo CLI allows inclusion of specific native dependencies and customizations.
+
+Initializing the Adobe Mobile SDK requires native code implementation. In Expo projects, Continuous Native Generation (CNG) is enabled by default, meaning the `Android` and `iOS` directories for native code and configuration are not included. To integrate the Mobile SDK, CNG must be disabled, and the `Android` and `iOS` directories must be generated and managed manually.
+
+## Installation
+
+To generate these directories, run 
+```bash
+npx expo prebuild
+```
+Alternatively, compile the app locally:
+```bash
+# Build your native Android project
+npx expo run:android
+# Build your native iOS project
+npx expo run:ios
+```
+
+## Install Adobe Mobile SDKs
+
+Refer to the [Installation Guide](../README.md#Installation) to install Adobe SDKs after the android and ios directories are generated.
+
+## Initialize Adobe Mobile SDKs
+Initializing the Mobile SDK involves implementing native code. The following code snippets show how to install and register the Mobile Core and Edge Network extensions. Documentation on how to initialize each extension can be found in _./packages/{extension}/README.md_.
+
+##### **iOS**
+
+Create a separate header and Objective-C file to handle the Adobe SDK setup, then import this file into the `AppDelegate` to initialize the SDK when the app launches.
+
+AdobeBridge.h
+```objective-c
+#import <UIKit/UIKit.h>
+#import <Foundation/Foundation.h>
+@interface AdobeBridge : NSObject
++ (void)configure: (UIApplicationState)state;
+@end
+```
+
+AdobeBridge.m
+```objective-c
+#import "AdobeBridge.h"
+#import <UIKit/UIKit.h>
+@import AEPCore;
+@import AEPEdgeIdentity;
+@import AEPEdgeConsent;
+@import AEPEdge;
+@import AEPIdentity;
+@implementation AdobeBridge
++ (void)configure: (UIApplicationState)appState
+{
+  [AEPMobileCore setLogLevel: AEPLogLevelDebug];
+  NSArray *extensionsToRegister = @[
+      AEPMobileEdgeIdentity.class,
+      AEPMobileEdgeConsent.class,
+      AEPMobileAssurance.class,
+      AEPMobileEdge.class,
+      AEPMobileIdentity.class,
+      AEPMobileLifecycle.class,
+      AEPMobileOptimize.class,
+      AEPMobileSignal.class,
+      AEPMobileUserProfile.class
+      ];
+      [AEPMobileCore registerExtensions:extensionsToRegister completion:^{
+        [AEPMobileCore configureWithAppId: @"KEY"];
+        if (appState != UIApplicationStateBackground) {
+            [AEPMobileCore lifecycleStart:@{@"": @""}];
+        }
+    }];
+}
+@end
+```
+
+AppDelegate.mm
+```objective-c
```

Comment:
```objective-c
#import "AdobeBridge.h" // add this import

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
  self.moduleName = @"main";

  // You can add your custom initial props in the dictionary below.
  // They will be passed down to the ViewController used by React Native.
  self.initialProps = @{};

  [AdobeBridge configure: application.applicationState];

  return [super application:application didFinishLaunchingWithOptions:launchOptions];
}

// Setup for Lifecycle Start when entering foreground
- (void)applicationWillEnterForeground:(UIApplication *)application{
  appState = application.applicationState;
  [AdobeBridge lifecycleStart];
}

// Setup for Lifecycle Start when entering background
- (void)applicationDidEnterBackground:(UIApplication *)application{
  appState = application.applicationState;
  [AdobeBridge lifecyclePause];
}

Reply:
`appState` need to declare this variable else everything is working fine @cacheung

Reply:
Updated.  @namArora3112